server/publications/collections/packages.js

```diff
@@ -69,10 +69,13 @@ function transform(doc, userId) {
 //
 //  Packages Publication
 //
-Meteor.publish("Packages", function (shopCursor) {
-  check(shopCursor, Match.Optional(Object));
+Meteor.publish("Packages", function (shopId) {
```

Comment:
Good question and nice find.

I should have mentioned that I checked on that, I looked at every instance where we subscribe to the Packages publication and not a single one used the shopCursor. I'm guessing it's an undeleted artifact of the way things used to be done, but not sure.

@aaronjudd any thoughts here?